### PR TITLE
Provide `Microsoft.AspNetCore.Authentication.Certificate` package README

### PR DESCRIPTION
**Pull request by MackinnonBuck**
Contributes to #48392

### CODE REVIEW

File: src/Security/Authentication/Certificate/src/PACKAGE.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+### Installation
+
+```shell
+dotnet add package Microsoft.AspNetCore.Authentication.Certificate
+```
+
+### Configuration
+
+1. Acquire an HTTPS certificate, apply it, and [configure your server](https://learn.microsoft.com/aspnet/core/security/authentication/certauth#configure-your-server-to-require-certificates) to require certificates
```
Comment (mkArtakMSFT) [inline src/Security/Authentication/Certificate/src/PACKAGE.md:17]: I think the `apply it` part is not clear. What does it mean?
Does it mean install the cert on your machine - trust it basically?

Comment (MackinnonBuck) [reply src/Security/Authentication/Certificate/src/PACKAGE.md:17]: Yeah, I'm guessing this refers to installing the cert on the machine. This line was taken directly from [these docs](https://learn.microsoft.com/aspnet/core/security/authentication/certauth#get-started).

Comment (mkArtakMSFT) [reply src/Security/Authentication/Certificate/src/PACKAGE.md:17]: Thanks Mackinnon. I think it's still worth clarifying it here. But up to you.

**Review by mkArtakMSFT — APPROVED** [review]

File: src/Security/Authentication/Certificate/src/PACKAGE.md (hunk 1 above)
Comment (MackinnonBuck) [reply src/Security/Authentication/Certificate/src/PACKAGE.md]: I've replaced "apply" with "install"

**Review by javiercn — APPROVED** [review]

**Pull request merged**